src/components/uku-chord/ukuChord.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { UkuChordStyled, UkuChordChartStyled, BarFretStyled, FretNumberStyled, FretStyled, StringStyled, UkuStringStatesStyled, UkuStringStateStyled, FingerNumberStyled } from '../uku-chord/ukuChord.styles'
+import { doNotPlayString, barFret } from '../../chords'
+import { isChordShortNameInKey, getKeyFromShortName, cleanNameForSounds } from '../../utils'
+import AudioPlayer from '../audio-player/audio-player'
+import soundFiles from './soundFiles'
+import FavouriteChordButton from '../favourite-chord-button/favourite-chord-button'
+import OutputMessage from '../output-message/output-message'
+import settings from '../../settings'
+
+const stringArray = [4, 3, 2, 1]
+
+const getIsChordHigh = frets => Object.keys(frets)[0] > 3
+
+const populateFretNumbers = frets => {
+  const firstFretNumber = parseInt(Object.keys(frets).shift())
+  return firstFretNumber > 3 ? [firstFretNumber, firstFretNumber+1, firstFretNumber+2, firstFretNumber+3] : [1, 2, 3, 4]
+}
+
+const getRomanNumeral = (chordShortName, selectedKeyShortName) => {
+  if (!selectedKeyShortName) {
+    return
+  }
+
+  const chordsInKey = getKeyFromShortName(selectedKeyShortName).chords
+
+  const chord = Object.entries(chordsInKey).find(([romanNumeral, chordShortNameUnderTest]) => chordShortNameUnderTest === chordShortName)
+
+  // This happens when you show every chord when filtering by key
+  if (!chord) {
+    return
+  }
+
+  return chord[0]
+}
+
+export const UkuChord = ({ selectedKeyShortName, fullName, shortName, alternativeShortName, strings = {}, frets = {} }) => {
```

Comment:
This component looks very similar to the guitar version. You could re-use them. Keep this component here but add a new prop to `Chord` called `isUkulele` and based off that prop, change the number of strings and such.

Reply:
Great, I will do this